packages/azureadrecon.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'AzureADRecon'
+$category = 'Cloud'
+
+$zipUrl = 'https://github.com/adrecon/AzureADRecon/archive/18410c1a630c085091ad15a768ac742fd2513902.zip'
+$zipSha256 = '51e0af14ae0e35300fe6fe8fa96ebbce86abb34a597fe81fd16b7358ebe13aae'
+
+VM-Install-Raw-GitHub-Repo $toolName $category $zipUrl $zipSha256
```

Comment:
This project has some prerequisites that we can probably handle, such as these:
```
Install-Module -Name AzureAD
Install-Module -Name Microsoft.Graph
```
I would add these commands in a try catch here so that we can get the prereq's installed

Reply:
Optionally we can look for chocolatey packages that do this already and add dependencies to the nuspec file